### PR TITLE
added terms

### PR DESCRIPTION
**Pull request by gizmo693**
added basic terms to the pdi charm

### CODE REVIEW

File: metrics.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ metrics:
 
 # transformation execution count
 
-# carte usage?
+# carte usage, hell yeah!
```
Comment (gizmo693) [inline metrics.yaml]: are you on drugs?

**Review by gizmo693 — COMMENTED** [review]
yes

File: metrics.yaml (hunk 1 above)
Comment (buggtb) [inline metrics.yaml]: drugs?

**Review by buggtb — CHANGES_REQUESTED** [review]
fix it

**Review by gizmo693 — COMMENTED** [review]
all fixed. sorry boss

**Review by buggtb — APPROVED** [review]

**Pull request merged**